### PR TITLE
making lite mode the default mode in the cfg

### PR DESCRIPTION
**Pull request by McEdward**
resolving issue #310

### CODE REVIEW

File: config.go [hunk 1]
```diff
@@ -89,7 +89,7 @@ type config struct {
 	DumpAllMPTix       bool   `long:"dumpallmptix" description:"Dump to file the fees of all the tickets in mempool."`
 	DBFileName         string `long:"dbfile" description:"SQLite DB file name (default is dcrdata.sqlt.db)."`
 
-	LiteMode bool   `short:"l" long:"lite" description:"Run in \"lite\" mode, using only SQLite."`
+	FullMode bool   `long:"full" description:"Run in \"Full Mode\" mode, using only SQLite."`
```
Comment (chappjc) [inline config.go]: description doesn't read right ("mode" repeated, and "using only SQLite" was for lite mode not full mode)

File: sample-dcrdata.conf [hunk 2]
```diff
@@ -37,7 +37,7 @@
 ;cachecontrol-maxage=86400
 
 ; lite mode uses only SQLite, no PostgreSQL, but certain features will not available
```
Comment (chappjc) [inline sample-dcrdata.conf]: description needs update

File: sample-dcrdata.conf [hunk 3]
```diff
@@ -37,7 +37,7 @@
...
-;lite=false
+;full=false
```
Comment (chappjc) [inline sample-dcrdata.conf]: "full" somehow feels more ambiguous than "lite".  Admittedly I don't have a better suggestion except perhaps "trickedout" or "fullbling" (just kidding)

**Comment by McEdward** [comment]
Current ready for review.

**Comment by chappjc** [comment]
Lgtm
Please rebase to resolve conflicts

**Comment by raedah** [comment]
tACK

**Pull request merged**